btcjson/chainsvrresults.go

```diff
@@ -855,3 +855,52 @@ type LoadWalletResult struct {
 type DumpWalletResult struct {
 	Filename string `json:"filename"`
 }
+
+// TestMempoolAcceptResult models the data from the testmempoolaccept command.
+// The result of the mempool acceptance test for each raw transaction in the
+// input array. Returns results for each transaction in the same order they
+// were passed in. Transactions that cannot be fully validated due to failures
+// in other transactions will not contain an 'allowed' result.
+type TestMempoolAcceptResult struct {
+	// The transaction hash in hex.
+	Txid string `json:"txid"`
```

Comment:
All the comment should start with the attr name to adhere to the existing `godoc` style.

Reply:
fixed